utils/model_uploader/update_pretrained_model_listing.py

```diff
@@ -0,0 +1,172 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+# This program is run by "Model Listing Uploading" workflow
+# (See model_listing_uploader.yml) to update pretrained_model_listing.json.
+
+import argparse
+import json
+import os
+from typing import Optional
+
+JSON_FILENAME = "pretrained_model_listing.json"
+JSON_DIRNAME = "utils/model_uploader/model_listing"
+PRETRAINED_MODEL_LISTING_JSON_FILEPATH = os.path.join(JSON_DIRNAME, JSON_FILENAME)
+PREFIX_SENTENCE_TRANSFORMER_FILEPATH = "ml-models/huggingface/sentence-transformers"
+TORCH_SCRIPT_FORMAT = "TORCH_SCRIPT"
+ONNX_FORMAT = "ONNX"
+TEMP_MODEL_PATH = "temp_model_path"
+TEST_SENTENCES = [
+    "First test sentence",
+    "Second test sentence",
+]
+
+
+def get_sentence_transformer_model_description(
+    config_folder_name: str, config_filepath: str
+) -> Optional[str]:
+    """
+    Get description of the pretrained sentence transformer model from config file
+
+    :param config_folder_name: Name of the local folder that stores config files (e.g. 'config_folder')
+    :type config_folder_name: string
+    :param config_filepath: Path to local config file
+    (e.g. 'sentence-transformers/all-MiniLM-L12-v2/2.0.0/onnx/config.json')
+    :type config_filepath: string
+    :return: Description of the model
+    :rtype: string or None
+    """
+    filepath = os.path.join(config_folder_name, config_filepath)
+    try:
+        with open(filepath, "r") as f:
+            model_config = json.load(f)
+    except Exception as e:
+        print(f"Cannot open {filepath} to get model description: {e}")
+        return None
+    if "description" in model_config:
+        return model_config["description"]
+    else:
+        return None
+
+
+def create_new_pretrained_model_listing(
```

Comment:
Is this possible to add unit test for this?

Reply:
@dhrubo-os Tests added. But I don't think it affects code coverage though.

Reply:
Just added the file to code coverage.